### DISCUSSION TITLE
Consider migration to Cooklang organisation

### DISCUSSION THREAD

**Discussion by dubadub** [cooklang/cooklang-obsidian#39] [General]
Hi there, 

Alexey from Cooklang is here. I'm wondering if you would consider moving that repo into Cooklang organisation? It seems that many users use Cooklang with Obsidian and we want to provide that plugin for them. We have a canonical parser which we plan to integrate for consistent experience, as well as add more features and provide support.

Regards
-Alexey.

**Comment by deathau** [comment]
I _have_ been neglecting this plugin, and would be more than happy to have it supported by those most passionate about Cooklang itself.

**Reply by deathau** [reply]
@dubadub, I don't have the necessary permission to transfer the repo to cooklang directly, but I have sent an invite to you to be a collaborator, which might give you access to do this? (I've never transferred a repo before)

**Reply by dubadub** [reply]
Thanks for your favour @deathau! I really appreciate that. Yeah, I find GitHub permissions cumbersome and not intuitive. Maybe you can try again in Settings: I added you as a Core team member in Cooklang organisation. That should be enough according to [their documentation](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#). Thanks!

**Reply by deathau** [reply]
Looks like that worked!
